.travis.yml

```diff
@@ -3,6 +3,9 @@ rvm:
   - 2.4.0
   - 2.3.3
   - 2.2.2
+gemfile:
+  - Gemfile
+  - Gemfile.sinatra1
```

Comment:
Do you mind renaming the file to `Gemfile.sinatra-1.x`? It's a minor change, but it feels clearer on what it's doing.